aip-console-tools-cli/src/main/java/com/castsoftware/aip/console/tools/commands/DeliverVersionCommand.java

```diff
@@ -184,7 +185,9 @@ protected Integer processCallCommand() throws Exception {
             return Constants.RETURN_APPLICATION_INFO_MISSING;
         }
 
-        log.info("Deliver version command has triggered with log output = '{}'", sharedOptions.isVerbose());
+        log.info("The Deliver version command has been triggered with:", sharedOptions.isVerbose());
```

Comment:
Are you sure that this call to log.info(...) should have "sharedOptions.isVerbose()" as second parameter ? beause I see no more a placeholder {} in the msg.